Core/DISET/private/Service.py

```diff
@@ -60,7 +61,16 @@ def __init__( self, serviceData ):
     if serviceData[ 'loadName' ] not in self._validNames:
       self._validNames.append( serviceData[ 'loadName' ] )
     self._cfg = ServiceConfiguration( list( self._validNames ) )
-    if serviceData[ 'standalone' ]:
+    # ElasticDB
+    self.activityMonitoring = gConfig.getValue("/DIRAC/ActivityMonitoring", "false").lower() in ("yes","true") 
```

Comment:
You cannot have this logic in the `__init__` because the remote configuration is not loaded yet. You have to do all this in the `initialize` method

Reply:
Sure, I will be making changes as we discussed yesterday and will push all the changes for further review.